arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md

```diff
@@ -0,0 +1,241 @@
+---
+title: 'How to Deploy a Rollup Chain Using the Orbit SDK'
+sidebar_label: 'Rollup Orbit Deployment'
+description: 'How to deploy a Rollup chain using the Orbit SDK '
+author: GreatSoshiant
+sme: GreatSoshiant
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 1
+---
+
+This document explains how to use the Orbit SDK to deploy a <a data-quicklook-from="arbitrum-rollup-chain">`Rollup Orbit chain`</a>.
+
+:::caution UNDER CONSTRUCTION
+
+This document is under construction and may change significantly as we incorporate [style guidance](/for-devs/contribute#document-type-conventions) and feedback from readers. Feel free to request specific clarifications by clicking the `Request an update` button at the top of this document.
+
+:::
+
+:::info
+
+See the ["create-rollup-eth" example](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/main/examples/create-rollup-eth/index.ts) in the Orbit SDK repository for additional guidance.
+
+:::
+
+The main benefit of the Orbit SDK lies in facilitating the deployment and fine-tuning of Orbit chains core Smart-Contracts. 
+
+These contracts are deployed on <a data-quicklook-from="parent-chain">`parent chains`</a>, they are:
+
+- <a data-quicklook-from="bridge">Bridge contracts</a>
```

Comment:
nitpick: I would change the order to "Rollup contracts", "Bridge contracts", "Contracts handling fraud proofs"